src/app/recents/recents.component.spec.ts

```diff
@@ -42,7 +43,11 @@ describe('RecentsComponent', () => {
         TabsModule.forRoot(),
         HttpClientTestingModule,
         NoopAnimationsModule,
-        BsDropdownModule.forRoot()
+        BsDropdownModule.forRoot(),
+        ToastrModule.forRoot({
```

Comment:
Optional: You could probably just do `ToastrModule.forRoot({})` in all the tests to simplify the module definition and reduce code.